src/App/App.js

```diff
@@ -8,17 +8,17 @@ import './App.css';
 import logo from '../RancidTomLogo.png';
 
 
-class App extends Component {
-  constructor() {
-    super();
+export class App extends Component {
+  constructor(props) {
+    super(props);
     this.state = {
       user: {},
       isOnHomePage: true
     }
   }
 
-  updateCurrentUser = (userInfo) => {
-    this.setState(userInfo)
+  updateCurrentUser = ({ user }) => {
```

Comment:
Per the project rubric, we are not supposed to be manipulating history. I have made an issue on the project board to refactor this. I think this should be an easy fix if we use link instead.